python/darpinstances/instance.py

```diff
@@ -284,30 +284,34 @@ def read_instance(filepath: Path, travel_time_provider: MatrixTravelTimeProvider
         while (line_string):
             line = line_string.split()
             request_id: int = int(index)
-            request_time: int = int(line[0]) / 1000
+            request_timestamp: int = int(line[0]) / 1000
+            request_datetime = datetime.fromtimestamp(request_timestamp)
             start_node = Node(int(line[1]))
             end_node = Node(int(line[2]))
             equipment = map_equipment_type(line[4]).value if(len(line) > 4) else 0
             min_travel_time = travel_time_provider.get_travel_time(start_node, end_node)
-            max_pickup_time = request_time + int(instance_config['max_prolongation'])
+            max_pickup_timestamp = request_timestamp + int(instance_config['max_prolongation'])
+            max_pickup_datetime = datetime.fromtimestamp(max_pickup_timestamp)
+            min_drop_off_timestamp = request_timestamp + min_travel_time
+            max_drop_off_timestamp = max_pickup_timestamp + min_travel_time
+            min_drop_off_datetime = datetime.fromtimestamp(min_drop_off_timestamp)
+            max_drop_off_datetime = datetime.fromtimestamp(max_drop_off_timestamp)
```

Comment:
same as above

Reply:
I understood that max_prolongation is a duration not a time, am I wrong?

Reply:
yes, but here you parse dates, not max prolongation

Reply:
Ok I get the point, it means just to adjust this section after the first comment change is aplied